bigtable/spark/src/main/scala/example/CopyTable.scala

```diff
@@ -0,0 +1,98 @@
+package example
+
+import org.apache.hadoop.hbase.spark.datasources.{HBaseSparkConf, HBaseTableCatalog}
+import org.apache.spark.sql.types.{IntegerType, StringType, StructField, StructType}
+
+object CopyTable extends App {
+
+  val appName = this.getClass.getSimpleName.replace("$", "")
+  println(s"$appName Spark application is starting up...")
+
+  val (projectId, instanceId, fromTable, toTable) = parse(args)
+  println(
+    s"""
+      |Parameters:
+      |projectId: $projectId
+      |instanceId: $instanceId
+      |copy from $fromTable to $toTable
+      |""".stripMargin)
+
+  import org.apache.spark.sql.SparkSession
+  val spark = SparkSession.builder().getOrCreate()
+  println(s"Spark version: ${spark.version}")
+
+  import com.google.cloud.bigtable.hbase.BigtableConfiguration
+  val conf = BigtableConfiguration.configure(projectId, instanceId)
+  import org.apache.hadoop.hbase.spark.HBaseContext
+  // Creating HBaseContext explicitly to use the conf above
+  // That's how to use command-line arguments for projectId and instanceId
+  // Otherwise, we'd have to use hbase-site.xml
+  // See HBaseSparkConf.USE_HBASECONTEXT option in hbase-connectors project
+  // https://github.com/apache/hbase-connectors/blob/rel/1.0.0/spark/hbase-spark/src/main/scala/org/apache/hadoop/hbase/spark/datasources/HBaseSparkConf.scala#L44
+  new HBaseContext(spark.sparkContext, conf)
+
+  def createCatalogJSON(table: String): String = {
+    s"""{
+       |"table":{"namespace":"default", "name":"$table", "tableCoder":"PrimitiveType"},
+       |"rowkey":"word",
+       |"columns":{
+       |  "word":{"cf":"rowkey", "col":"word", "type":"string"},
+       |  "count":{"cf":"cf", "col":"Count", "type":"int"}
+       |}
+       |}""".stripMargin
+  }
+
+  // The HBaseTableCatalog options are described in the sources themselves only
+  // https://github.com/apache/hbase-connectors/blob/rel/1.0.0/spark/hbase-spark/src/main/scala/org/apache/hadoop/hbase/spark/datasources/HBaseSparkConf.scala
+
+  println(s"Loading records from $fromTable")
```

Comment:
I'd add some additional comments here as to what we're doing (specifically around `createCatalogJSON`)